services/vision/mlvision/classifier.go

```diff
@@ -22,8 +22,14 @@ func attemptToBuildClassifier(mlm mlmodel.Service) (classification.Classifier, e
 
 	// Set up input type, height, width, and labels
 	var inHeight, inWidth uint
+	if len(md.Inputs) < 1 {
+		return nil, errors.New("could not get information about first input tensor")
+	}
 	inType := md.Inputs[0].DataType
 	labels := getLabelsFromMetadata(md)
+	if len(md.Inputs[0].Shape) < 4 {
+		return nil, errors.New("could not retrieve dimensions for input tensor")
```

Comment:
This error isn't that the it couldn't get the shape of the input tensor, but that the dimension of input was less than the expected 4, right?